File: Domovoy/ClientApp/src/api/models/ConstructionCompany.ts
/* istanbul ignore file */
/* tslint:disable */
/* eslint-disable */

import type { ApplicationUser } from './ApplicationUser';
import type { ResidentialComplex } from './ResidentialComplex';

export type ConstructionCompany = {
    id?: number;
    name?: string | null;
    employees?: Array<ApplicationUser> | null;
    residentialComplexes?: Array<ResidentialComplex> | null;
};

File: Domovoy/Models/ConstructionCompany.cs
namespace Domovoy.Models;

public class ConstructionCompany
{
    public int Id { get; set; }
    public string Name { get; set; }
    public List<ApplicationUser> Employees { get; set; }
    /// <summary>
    /// If ConstructionCompanyAdmin
    /// </summary>
    public List<ApartmentHouse> ApartmentHouses { get; set; }
}